README.md

```diff
@@ -1 +1,13 @@
-# fedora-coreos-metrics-client
\ No newline at end of file
+# fedora-coreos-metrics-client
+
```

Comment:
Would be good to have a short description here of what the goal of this project is.

Reply:
Thanks, added this! Also just added in some documentation on configuring it.